src/tedpy/dataclasses.py

```diff
@@ -39,7 +39,7 @@ class SystemType(Enum):
 
     NET = 0
     NET_GEN = 1
-    NET_LOAD = 2
+    GEN_LOAD = 2
```

Comment:
I did some digging on this one because I was wondering how I made such a bad typo. It turns out the actual interface spec calls out "NET+LOAD" as the proper name. I got the LOAD_GEN stuff from a forum post on the TED forums mixing TED5000 threads in TED6000 code. Probably should revert this back to "NET_LOAD"

Reply:
I like the `LOAD_GEN` name better though, because it seems in that mode you can only configure MTUs as GEN or LOAD, not NET. So I don't know why they chose the name NET+LOAD.

Reply:
Agreed the name doesn't make much sense. I'm fine with keeping it as is.